frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeaderView.tsx

```diff
@@ -112,52 +113,63 @@ export function DashboardHeaderComponent({
+            data-testid="fixed-width-dashboard-header"
+            isNavBarOpen={isNavBarOpen}
+            isFixedWidth={dashboard?.width === "fixed"}
           >
-            <HeaderCaptionContainer>
-              <HeaderCaption
-                key={dashboard.name}
-                initialValue={dashboard.name}
-                placeholder={t`Add title`}
-                isDisabled={!dashboard.can_write}
-                data-testid="dashboard-name-heading"
-                onChange={handleUpdateCaption}
-              />
-              <PLUGIN_COLLECTION_COMPONENTS.CollectionInstanceAnalyticsIcon
-                color={color("brand")}
-                collection={collection}
-                entity="dashboard"
-              />
-            </HeaderCaptionContainer>
-            <HeaderBadges>
-              {isLastEditInfoVisible && (
-                <HeaderLastEditInfoLabel
-                  item={dashboard}
-                  onClick={onLastEditInfoClick}
-                  className=""
+            <HeaderContent
+              role="heading"
+              hasSubHeader
+              showSubHeader={showSubHeader}
+            >
+              <HeaderCaptionContainer>
+                <HeaderCaption
+                  key={dashboard.name}
+                  initialValue={dashboard.name}
+                  placeholder={t`Add title`}
+                  isDisabled={!dashboard.can_write}
+                  data-testid="dashboard-name-heading"
+                  onChange={handleUpdateCaption}
                 />
-              )}
-            </HeaderBadges>
-          </HeaderContent>
+                <PLUGIN_COLLECTION_COMPONENTS.CollectionInstanceAnalyticsIcon
+                  color={color("brand")}
+                  collection={collection}
+                  entity="dashboard"
+                />
+              </HeaderCaptionContainer>
+              <HeaderBadges>
+                {isLastEditInfoVisible && (
+                  <HeaderLastEditInfoLabel
+                    item={dashboard}
+                    onClick={onLastEditInfoClick}
+                    className=""
+                  />
+                )}
+              </HeaderBadges>
+            </HeaderContent>
 
-          <HeaderButtonsContainer isNavBarOpen={isNavBarOpen}>
-            {_headerButtons}
-          </HeaderButtonsContainer>
+            <HeaderButtonsContainer isNavBarOpen={isNavBarOpen}>
+              {_headerButtons}
+            </HeaderButtonsContainer>
+          </StyledFixedWidthContainer>
         </HeaderRow>
-        <HeaderRow isNavBarOpen={isNavBarOpen}>
-          <DashboardTabs
-            dashboardId={dashboard.id}
-            location={location}
-            isEditing={isEditing}
-          />
+        <HeaderRow>
+          <StyledFixedWidthContainer
+            data-testid="fixed-width-dashboard-tabs"
+            isNavBarOpen={isNavBarOpen}
+            isFixedWidth={dashboard?.width === "fixed"}
+          >
```

Comment:
Should `isNavBarOpen` be passed to `HeaderRow` instead of `StyledFixedWidthContainer` ?